packages/atlas-core/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [3.9.1] Atlas Core - 2023-4-26
```

Comment:
This line is added automatically. Treat `## [Unreleased]` section as a version title, put the `### Changed` section under it, and that is it, automation takes care of the rest.